src/modules/LanUdevAuto.rb

```diff
@@ -180,7 +180,7 @@ def Import(settings)
           createUdevFromIfaceName(Ops.get_list(settings, "interfaces", []))
         )
       else
-        @udev_rules = Ops.get_list(settings, "net-udev", [])
+        @udev_rules = settings["net-udev"] || []
```

Comment:
note that the fix was already present in the master, so here it disappears in the merge.